ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -496,13 +496,13 @@ FEDataManager::getDefaultSpreadSpec() const
 const std::vector<std::vector<Elem*> >&
 FEDataManager::getActivePatchElementMap() const
 {
-    return d_active_patch_elem_map;
+    return d_active_patch_elem_map.back();
```

Comment:
While not ideal this is necessary to keep IBFEMethod and IBFESurfaceMethod happy at the moment.

Reply:
I was going to comment on that. :-D

Reply:
I don't want to break the IIM branch even more badly than I already have